tests/app/celery/test_contact_information_tasks.py

```diff
@@ -58,8 +51,11 @@ def test_should_get_email_address_and_update_notification(client, mocker, notifi
     assert notification.to == 'test@test.org'
 
 
-def test_should_get_phone_number_and_update_notification(client, mocker, notification):
-    notification.notification_type = SMS_TYPE
+def test_should_get_phone_number_and_update_notification(client, mocker, sample_notification):
```

Comment:
"" test naming

Reply:
We are changing our approach to testing.